src/components/app/details/triggerView/types.ts

```diff
@@ -83,11 +83,14 @@ export interface CIMaterialProps extends RouteComponentProps<CIMaterialRouterPro
     getWorkflows: () => void
     loader: boolean
     setLoader: (isLoading) => void
+    isFirstTrigger?: boolean
+    isCacheAvailable?: boolean
 }
 
 export interface CIMaterialState {
     regexValue: Record<number, { value: string; isInvalid: boolean }>
     selectedCIPipeline?: any
+    isBobStorageConfigured?: boolean
```

Comment:
Seems typo - should be `isBlobStorageConfigured`

Reply:
done